ingest/clusters.py

```diff
@@ -73,7 +74,27 @@ def __init__(
     def validate(self):
         """ Runs all validation checks
         """
-        return all([self.is_valid_format(), self.validate_numeric_annots()])
+        return all(
+            [
+                self.is_valid_format(),
+                self.validate_numeric_annots(),
+                self.require_X_Y_not_nan(),
+            ]
+        )
+
+    def require_X_Y_not_nan(self):
+        """
+        X Y and Z are expected to be fully populated in cluster files
+        Fail coordinate data columns that have nan values
```

Comment:
tiny NIT but wondering if this should be NaN to keep the capitalization consistent?

Reply:
fixed in c7c3f38